manim/cli/new/group.py

```diff
@@ -0,0 +1,207 @@
+import configparser
+from pathlib import Path
+
+import click
+
+from ... import console
+from ...constants import CONTEXT_SETTINGS, EPILOG, QUALITIES
+from ...utils.file_ops import (
+    add_import_statement,
+    copy_template_files,
+    get_template_names,
+    get_template_path,
+)
+
+CFG_DEFAULTS = {
+    "frame_rate": 30,
+    "background_color": "BLACK",
+    "background_opacity": 1,
+    "scene_names": "Default",
+    "resolution": (854, 480),
+}
+
+
+# select_resolution() called inside project command
+def select_resolution():
+    """Prompts input of type click.Choice from user. Presents options from QUALITIES constant.
+
+    Returns
+    -------
+        :class:`tuple`
+            Tuple containing height and width.
+    """
+    resolution_options = []
+    for quality in QUALITIES.items():
+        resolution_options.append(
+            (quality[1]["pixel_height"], quality[1]["pixel_width"])
+        )
+    resolution_options.pop()
+    choice = click.prompt(
+        "\nSelect resolution:\n",
+        type=click.Choice([f"{i[0]}p" for i in resolution_options]),
+        show_default=False,
+        default="480p",
+    )
+    return [res for res in resolution_options if f"{res[0]}p" == choice][0]
+
+
+def update_cfg(cfg_dict, project_cfg_path):
+    """Updates the manim.cfg file after reading it from the project_cfg_path.
+
+    Parameters
+    ----------
+    cfg : :class:`dict`
+        values used to update manim.cfg found project_cfg_path.
+    project_cfg_path : :class:`Path`
+        Path of manim.cfg file.
+    """
+    config = configparser.ConfigParser()
+    config.read(project_cfg_path)
+    cli_config = config["CLI"]
+    for key, value in cfg_dict.items():
+        if key == "resolution":
+            cli_config["pixel_height"] = str(value[0])
+            cli_config["pixel_width"] = str(value[1])
+        else:
+            cli_config[key] = str(value)
+
+    with open(project_cfg_path, "w") as conf:
+        config.write(conf)
+
+
+@click.command(
+    context_settings=CONTEXT_SETTINGS,
+    no_args_is_help=False,
+    epilog=EPILOG,
+    help="Create new projects.",
+)
+@click.argument("project_name", type=Path, required=False)
+@click.argument("template_name", required=False)
+@click.option(
+    "-d",
+    "--default",
+    "default_settings",
+    is_flag=True,
+    help="Default settings for project creation.",
+    nargs=1,
+)
+def project(default_settings, **args):
+    """Project subcommand creates a project inside of a 'new folder'. The name of the 'new folder' is passed as the first argument to the command.
```

Comment:
I'm not sure why `'new folder'` is in quotes here. It seems like this should be `'project_name'` instead since that Path variable would be the folder created?

Reply:
yes totally. my bad sorry. I changed it to project_name. I am not very good at naming stuff.